build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'io.franzbecker.gradle-lombok' version '1.11'
+    id 'java'
+}
+group 'io.leonis'
+version '1.0-SNAPSHOT'
+lombok {
+    version = "1.16.20"
+    sha256 = ""
+}
+sourceCompatibility = 1.8
+repositories {
+    mavenCentral()
+}
+dependencies {
+    compile 'io.reactivex.rxjava2:rxjava:2.1.9'
```

Comment:
I asked you in the chat earlier to use `compile 'io.reactivex.rxjava2:rxjava:2.+'` to make this just work with any rxjava v2. Since we're just relying on Function instead of actual impls.